test/unit-test/test_btc_sign.c

```diff
@@ -88,7 +88,6 @@ static void _test_btc_sign_init(void** state)
         .version = 1,
         .num_inputs = 1,
         .num_outputs = 1,
-        .locktime = 0,
```

Comment:
why remove this?

Reply:
It's only used in that single test case. That said, it would be much better to actually repair whatever is breaking the test currently.

Reply:
In that case please restore it. this is just the struct initialization for the tx that is used in all tests. Sure, `0` is the default, but I prefer explicitness.

Reply:
Done.